modules/backend/lang/en/lang.php

```diff
@@ -141,7 +141,7 @@
             'name_field' => 'Name',
             'description_field' => 'Description',
             'is_new_user_default_field_label' => 'Default group',
-            'is_new_user_default_field_comment' => 'Add new administrators to this group by default',
+            'is_new_user_default_field_comment' => 'Add new administrators to this group by default.',
```

Comment:
The rest of the changes are fine with me, however I just don't think we're consistent enough with which translation strings end with a period vs which don't to make those changes. I'm thinking either we come up with a consistent approach to that and implement it for all the strings within the English language files or we just don't bother and leave it as is for now. What are your thoughts on that?